### DISCUSSION TITLE
SD card error when going through menus & testing 1k resistor error HW3

### DISCUSSION THREAD

**Discussion by Soulboogie** [sanni/cartreader#833] [Need help]
Hello, I recently received my cartridge reader from Save the Hero Builders, and I have a couple of questions.

During the self-test (eep), I encountered a failed 1k resistor test. However, I'm unable to consistently replicate the failure as it sometimes passes without any changes.

Additionally, I've been facing an SD card error while navigating the menus, especially when returning to the main menu. To resolve this, I need to remove and reinsert the SD card, and then press the button to continue.

I tried formatting the SD card to fat32, but unfortunately, it didn't make any difference.

**Comment by Soulboogie** [comment]
Replacing the SD card with another I had laying around fix the SD error, but stilling inconsistent passes on the 1k resistor test.

**Reply by SAVE-THE-HERO** [reply]
It can be visually confirmed from back side.(tester is required for electrical checks.
 
![IMG_8200](https://github.com/sanni/cartreader/assets/89830526/7b0a961c-d380-417f-9c32-428bdbb2218d)

**Reply by 7ank0v1c** [reply]
I can confirm that mine did the same with the 1K resistor

**Reply by SAVE-THE-HERO** [reply]
I forgot to test it myself.
Thank you @7ank0v1c
I got the same 1K error on mine.
The first time I got an error, but I can't confirm it the next time.
I don't think the test itself is for v3.
However, the fact that the error cannot be confirmed is a mystery.

**Reply by SAVE-THE-HERO** [reply]
The 1K resistor is needed for reading/writing the N64 eeprom and the Controller Pak saves
If you can not  N64 eeprom and the Controller Pak saves pls give me mail/

**Comment by SAVE-THE-HERO** [comment]
Hello.
I am the seller of Save the Hero Builders.
If you have any questions regarding product defects, please contact me at my email.
savetheherobuilders@gmail.com
---
The SD card included for that test may be defective.
We are very sorry, but please try another SD card that you have.
We test all slots for dump before shipping.
The SD card is the last one to be replaced, so sometimes there are defects mixed in.

I think 1k resistor test does not work well in v3?
If you have a title that you can't Dump, please email me.

Thank you

**Comment by Androxilogin** [comment]
> received my cartridge reader from Save the Hero Builders

Well there's your problem, right there. I can't believe how many people bought from this shyster. lol. Good luck to ya.

**Reply by SAVE-THE-HERO** [reply]
Oh!
You are in here!
It's been a long time.
I was a little worried when you didn't come in.
If you are doing well, that's fine with me.
It's hot in the summer now,I hope you are well.
Best regards.

**Reply by 7ank0v1c** [reply]
That’s a little rude @Androxilogin, flip. Chill dude chill

**Reply by 7ank0v1c** [reply]
For you info @Androxilogin some people buy because they don’t have time to build one themselves. I didn’t, neither do I have confidence I wouldnt mess it up. There is a market for people like me… please no need for the hate

**Comment by Androxilogin** [comment]
> That’s a little rude @Androxilogin, flip. Chill dude chill

Idk, creating a kickstarter without permission for an open source project that already existed then continuing to do it after devs responded negatively to it was pretty rude, I thought. Not to mention sending a wave of problems to the github over the years for free customer support for your own mishaps while assembling. Amongst other things previously expressed here.. But hey, feel how you will.

**Reply by SAVE-THE-HERO** [reply]
I did not intend to interfere with your sale.
I am really sorry.

**Reply by Androxilogin** [reply]
You missed the message. Sales for other people was never a problem, inexperienced self indulgence and your own greed at others expense was the issue. This has all been previously addressed. Unfortunately your customers are still unaware of the kindness of the community you've taken advantage of, which is sad. But the contributors still graciously continue on for the sake of the project and accept it, the noble lot. Just wanted to drop a gentle reminder.

**Reply by 7ank0v1c** [reply]
If I may weigh in on this a little, I do get the sentiment here and can understand why the devs might not like it. It makes sense, making money on something that is available to all is bad. Things is though, it isn’t actually available to all is it…?

It's only available to those who have tools, expertise and time on their hands to build one.

It is open source after all, there aren’t rules that someone can’t do this. What’s more from what I have known since I have bought one (I can’t speak for prior to October 2022), save the hero has always acknowledged this project on his website etc…

Besides, I now have a great unit which looks awesome asctetically which I would not have if I did take the time and energy etc… to build for myself.

**Reply by 7ank0v1c** [reply]
@Androxilogin 

![IMG_8318](https://github.com/sanni/cartreader/assets/40895945/6809577d-5e96-4708-8f80-d22afd377e24)

**Reply by 7ank0v1c** [reply]
I also am very greatful for the whole Sanni team and community, I now have an awesome device which is way better than the antcient retrode 2 I had from way back. You guys are awesome and keep bringing great new updates and even adapters 😊

**Reply by 7ank0v1c** [reply]
Both save the hero and the whole of the Sanni team has also enabled me now to be an active member in the preservation community too. I wouldn’t be with out this

**Reply by 7ank0v1c** [reply]
Anyways that’s my thoughts, everyone is entitled to their own. Just wanted to say my bit, back to you @Androxilogin 😜

**Reply by Androxilogin** [reply]
This was all prior. 2019-2020.. Mass producing off of someone else without first asking permission and then asking and them saying they don't like the idea and doing it anyway while pretending to be the creator speaking with YouTube creators.. There are many layers to this tale. 

I'd be pretty pissed paying for a piece of acrylic stapled to a block of wood for a case to be honest with you. There's been a full case with buttons rather than that off breed dust invitation of a mess available for years. It's a shame to only get half a case with your expensive purchase. 

A huge bravo to the actual developers, but absolutely no respect for the get rich quick grifter. Without him, there would, were, and surely still are other affordable options from people who actually knew how the hardware works and didn't somehow burn their hands with a soldering iron repeatedly while putting them together and post photos on social networking as if it were some sort of accomplishment. Just look at this post- the 1k resistor was added to the board without even knowing what it is for. People are surely happy to possess the functionality, but there could have been other options built with more reliable parts by more experienced people referred to specifically by the ones who put their all into this project and actually make sure it works 100% before peddling lemons. But anyway, my comment wasn't meant to dwell into all of the details of this shameful scheme. The developers deserve all of the recognition, the opportunist deserves no praise whatsoever.

**Reply by SAVE-THE-HERO** [reply]
The 1K resistor is needed for reading/writing the N64 eeprom and the Controller Pak saves

@Androxilogin
You are free to dislike me, but I do not wish non-constructive discussions in this place.
Take care of yourself and stay healthy.
Thank you

**Reply by Soulboogie** [reply]
Thank you for the response if I have any further questions I'll make sure to send it your support email. Sorry for any issues my post has cause.

**Reply by SAVE-THE-HERO** [reply]
There is nothing wrong with your post.
Im sorry too.
and Thank you so much.

**Reply by Ancyker** [reply]
I'm a bit confused, I was under the impression @sanni was fine with it as I've seen posts by them saying as much.

Also, the license allows for what STH is doing. I mean, I sell cart readers too. I didn't start a Kickstarter but I also have the funds to invest so I don't really need to, plus I dislike using Kickstarter in general due to the association of it with [vaporware](https://en.wikipedia.org/wiki/Vaporware)/scam products.

**Reply by Androxilogin** [reply]
I could go digging up all of the discussion during this period, but it's best to just leave it. You can find it under their old name on here if you really want to look into it all. My comment was meant to be a low-scale and ignored, only in reference to the history, not full-on stir up the past. It was ultimately brushed aside by the ones doing the work so that's all that really matters but in my opinion, was a pretty low thing to do which I feel people should also be aware of. I had a lot of things I wanted to contribute during that time but this whole situation caused me to completely discard everything I had planned on sharing. So myself, I can't help but hold a grudge. 

Don't get me wrong, someone just doing their thing, having a good time making and selling as it was. It takes time and it was honest. And it was a great time to be here. I was very cautious about getting permissions for things prior to doing them here and to see someone just walk in and step all over a respectful way of doing things didn't sit right with me, personally. I don't want to speak for the ones actually doing everything here, but for one person to step in and start mass producing, along the comments of entitlement that were made during that time discarded as if misunderstood as the result of a language barrier on many occasions.. I didn't want to contribute to any part of that sort of thing no matter how miniscule. It was never my project or hard work or rules to make to begin with, it was just selfish and dirty without any remorse. But it was also accepted in such a generous community. I feel it would have made waves regardless, only more signifcantly and beneficial to those here who deserve the credit.

**Reply by Ancyker** [reply]
Well, that predates my involvement. I did search for things related to STH when I first found the project and only found 1-2 posts where Sanni mentioned it and I read them as defending STH. I'd be interested in reading these posts if you don't mind digging them up as I was unable to through my searching. If you'd rather not drag this up on here, which is totally understandable, you can PM me them on Discord.

With that said, regardless of the past, I feel I should mention that if not for STH being featured on LTT I likely never would have found the project, or if I did would have much later. I had searched for a dumper for N64 and came up empty-handed as all the ones I found were no longer in production or were only planned/crowdfunding. When I saw the LTT video I jumped on here and well, here we are...

I can't speak for everyone but I personally feel the more people with access to the cart reader the better. I'm not even mad about it showing up on Aliexpress, at least in terms of it being for sale. The listing stole Bonzo's pictures though and that's not cool. I've started watermarking mine as a result.

**Comment by Ancyker** [comment]
I think the 1K resistor test was failing because you did not have the EEP switch turned on. I've [added a message to do this](https://github.com/sanni/cartreader/commit/6733c9f57c03b70a6062afe0c012c9cc0941e335) to the test so that people know. With the switch in the off position, the pin is left floating so the result will be random.

**Reply by PsyK0p4T** [reply]
Behavior confirmed on a hw3 vanilla (not STH). 
Resistor test fails if EEP switch is off.
Your warning message displayed well and the test is ok with EEP on.

**Reply by 7ank0v1c** [reply]
This was confirmed for me too on a STH unit, same behaviour. It does seem though that it’s a HW3 related thing, not so much a STH thing due to the frequency and @PsyK0p4T results

**Reply by SAVE-THE-HERO** [reply]
I got same confirmation too.
Thank you so much.
